client/webpack.server.rails.config.js

```diff
@@ -8,13 +10,17 @@ module.exports = {
   output: {
     filename: 'server-bundle.js',
     path: '../app/assets/javascripts/generated',
-
-    // CRITICAL for enabling Rails to find the globally exposed variables.
-    libaryTarget: 'this',
   },
   resolve: {
     extensions: ['', '.webpack.js', '.web.js', '.js', '.jsx', 'config.js'],
   },
+  plugins: [
```

Comment:
`client.rails.config` inherits too much client stuff from `client.base.config`, so I'd keep it as is for readability sake until there will be more commons.